Makefile

```diff
@@ -104,17 +93,9 @@ STYLEFILES=$(PYFILES) $(BIN_FILES)
 
 .DEFAULT_GOAL := build
 
-# Install doesn't perform a build if it doesn't have too.  Best to clean out
-# any cruft so developers don't end up install old builds.
-ifeq ($(WITH_SUBMAN_GUI),true)
-    build: rhsmcertd rhsm-icon
-        EXCLUDE_PACKAGES:="$(EXCLUDE_PACKAGES)" $(PYTHON) ./setup.py clean --all
-        EXCLUDE_PACKAGES:="$(EXCLUDE_PACKAGES)" $(PYTHON) ./setup.py build --quiet --gtk-version=$(GTK_VERSION) --rpm-version=$(VERSION)
-else
-    build: rhsmcertd
```

Comment:
I'm sure that we still want to build rhsmcertd!

Reply:
Are you reading this change correctly? The rhsmcertd is kept there, it was just lifted from the if-else block and the `--gtk-version` argument has been removed.

Reply:
Oh, I can see it now. You are right.